### DISCUSSION TITLE
Case Study: Formatting Dates in PostgreSQL with Vultr

### DISCUSSION THREAD

**Discussion by new-php** [pgjdbc/r2dbc-postgresql#671] [Ideas]
**The Problem:** I was struggling with [date formatting in PostgreSQL](https://docs.vultr.com/how-to-use-the-postgresql-date-data-type) for my project. I needed the dates in specific formats like DD/MM/YYYY for reports and YYYY-MM-DD for APIs, but I wasn't sure how to implement it correctly. I also needed to make sure everything worked seamlessly with my cloud infrastructure, but the setup was overwhelming.

**The Solution:** I decided to use Vultr’s managed PostgreSQL service, which made it incredibly easy to focus on my work without dealing with complex configurations. With Vultr, I didn't have to worry about managing the database server myself.

Once I set up the PostgreSQL instance on Vultr, I quickly discovered the power of the` TO_CHAR() `function to format dates as needed. It was straightforward: `I used SELECT TO_CHAR(NOW(), 'DD/MM/YYYY')` to get the current date in the desired format. Vultr’s environment ensured my PostgreSQL instance was up and running without any hiccups, and I could focus on building my application.

**The Result:** Thanks to Vultr's simple setup and seamless [PostgreSQL management](https://docs.vultr.com/how-to-manage-users-in-vultr-managed-databases-for-postgresql), I saved time and got my date formatting issues resolved effortlessly. Vultr took care of the infrastructure, allowing me to focus entirely on my development and database queries.

**Comment by mp911de** [comment]
This repository is related to driver issues between a reactive application and the Postgres database. Your issue is related to SQL and formatting, nothing we can really help with.
